node-operators/block-producers/onboarding/account-setup.md

```diff
@@ -0,0 +1,152 @@
+
+Take note of the operation and the session number at which you submitted the request, as you'll need both values to execute the pending request.
+
+You can run another query from the **Chain state** page to check the current session. To do so, you can:
+
+1. Select the **session** module and the **currentIndex** query
+2. Click the **+** button next to the extrinsic field
+3. The current session will be displayed at the bottom of the page
+
+![Query the current session index on Polkadot.js Apps](/images/node-operators/block-producers/onboarding/account-setup/setup-5.webp)
+
+If the request can be executed, select **Extrinsics** from the **Developer** dropdown and take the following steps:
+
+1. Select the account from which you want to send the extrinsic
+2. Select the **pooledStaking** module and the **executePendingOperations** extrinsic
+3. For **delegator**, enter your account, which is the same account you sent the self-delegate request from
+4. For **operation**, select the type of operation to execute. This should be either **JoiningAutoCompounding** or **JoiningManualRewards**, depending on the target pool you selected at the time of submitting the self-delegation request
+5. For **candidate**, enter the same account as you did in the **delegator** field
+6. For **at**, enter the session key at which you submitted the delegate request
+7. Click **Submit Transaction** and sign and send the transaction from your wallet
+
+![Create and submit an extrinsic to execute the pending self-delegation request on Polkadot.js Apps](/images/node-operators/block-producers/onboarding/account-setup/setup-4.webp)
+
+Now, you have completed all of the necessary account setup to be eligible to produce blocks!
+
+## Verify That Your Account Is in the List of Eligible Candidates {: #verify }
+
+If you've followed all of the steps in this guide and have fully synced your block-producing node, you are now eligible to produce blocks. To verify that you are in the list of eligible candidates, you can go to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://fraa-dancebox-rpc.a.dancebox.tanssi.network#/extrinsics){target=\_blank}, click on the **Developer** tab, select **Chain state** from the dropdown, and take the following steps:
+
+1. Select the **pooledStaking** module and the **sortedEligibleCandidates** query
+2. Click the **+** button next to the extrinsic field
+3. A list of the eligible candidates and their stake will be displayed at the bottom of the page. You can search for your address to ensure you are eligible to produce blocks
+
+![Query the current list of eligible candidates on Polkadot.js Apps](/images/node-operators/block-producers/onboarding/account-setup/setup-4.webp)
+
+Remember that you'll need to be in the top {{ networks.dancebox.staking.top_eligible_candidates }} candidates by total stake to produce blocks.
```

Comment:
It's related to a previous comment. Not sure if that top_eligible_candidates number is static